Makefile

```diff
@@ -327,7 +327,7 @@ ctl:
 # A special target for testing only "coprocessor::dag::expr"
 # per https://github.com/tikv/tikv/pull/3280
 expression: format clippy
-	RUST_BACKTRACE=1 cargo test --features "${ENABLE_FEATURES}" --no-default-features --package tidb_query "expr" -- --nocapture
+	RUST_BACKTRACE=1 cargo test --features "${ENABLE_FEATURES}" --no-default-features --package "expr" -- --nocapture
```

Comment:
The whole command should not be working any more... I think we can just remove the whole expression command...